### PR TITLE
v4.6 changelog

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by esethna** [comment]
Made a few tweaks. Looks good other than the XXX marks that still need to be addressed

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,151 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+
+// XXX @derrick can you help confirm this section?
+
+**IncomingWebhooks Table:**
+- Renamed `PostUsername` column to `Username`.
+- Renamed `PostIconURL` column to `IconURL`.
+
+### WebSocket Changes
+
+// XXX @derrick can you help with this section?
```
Comment (crspeller) [inline source/administration/changelog.md]: No websocket changes.

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,151 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Added `UnregisterCommand` to unregister a command previously registered via `RegisterCommand`.
+- Added `GetChannelMember` to get a channel membership for a user.
+
+#### Plugin Hook Changes (Beta)
+
+- Added `ExecuteCommand` hook to execute a command that was previously registered via the `RegisterCommand` plugin API.
+
+### Database Changes
+
+// XXX @derrick can you help confirm this section?
```
Comment (crspeller) [inline source/administration/changelog.md]: These are the only DB changes.

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,151 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- Under `LdapSettings` in `config.json`:
+  - Added `"LoginButtonColor": ""`, `"LoginButtonBorderColor": ""` and `"LoginButtonTextColor": ""` to set the style of the LDAP login button for white labelling purposes.
+- Under `SamlSettings` in `config.json`:
+  - Added `"LoginButtonColor": ""`, `"LoginButtonBorderColor": ""` and `"LoginButtonTextColor": ""` to set the style of the SAML login button for white labelling purposes.
+
+### API Changes
+
+// XXX @derrick can you help with this section?
+
+- It is recommended that any new integrations use API v4 endpoints. For more details, and for a complete list of available endpoints, see [https://api.mattermost.com/](https://api.mattermost.com/).
```
Comment (crspeller) [inline source/administration/changelog.md]: Not recommended, "required".

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,151 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
...
+- All API v3 endpoints are now deprecated, and scheduled for removal in Mattermost v5.0.
+
+#### RESTful API v4 Changes
+
+- Added `/users/{user_id}/auth` to update a user's authentication method. This can be used to change them to/from LDAP authentication, for example.
```
Comment (crspeller) [inline source/administration/changelog.md]: Looks like this is everything.

**Pull request merged**